packages/palette/src/elements/Text/Text.tsx

```diff
@@ -65,37 +65,39 @@ export type TextProps = BaseTextProps &
 
 /** Text */
 export const Text = styled(Box)<TextProps>`
-  ${(props) => {
-    return getThemeConfig(props, {
-      v2: css`
-        ${variant({ variants: V2_TEXT_VARIANTS.small })}
-        ${textMixin}
+  && {
```

Comment:
This increases specificity by doubling up the class names. `.Text` would become `.Text.Text`